components/raftstore/src/store/peer.rs

```diff
@@ -1072,9 +1073,10 @@ where
         T: Transport,
         I: IntoIterator<Item = eraftpb::Message>,
     {
+        let disk_status = disk::get_disk_status();
         for msg in msgs {
             let msg_type = msg.get_msg_type();
-            let i = self.send_raft_message(msg, trans) as usize;
+            let i = self.send_raft_message(msg, trans, disk_status) as usize;
```

Comment:
It's wrong. It should be the `DiskFullOpt` of the carried commands instead of the peer's disk status. And please consider this situation: proposing a drop table operation, and then proposing a large normal entry on the same region. `RaftMessage` should contains the information about `Vec<(log_index, DiskFullOpt)>`.